distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1086,15 +1089,16 @@ type Table
 
        ? Result Ordering
 
-         Rows in the result are first ordered by the order of the corresponding
-         rows from the left table and then the order of rows from the right
-         table. This applies only if the order of the rows was specified (for
-         example, by sorting the table; in-memory tables will keep the memory
-         layout order while for database tables the order may be unspecified).
+         The ordering of rows in the resulting table is not specified.
```

Comment:
I assume we don't want to apply an ordering by default since it's much more expensive than it is for in-mem.

Reply:
Yeah, although IMO we should have the same docs for both variants so maybe let's just say that in in-memory it is as above and in Database unspecified.

Reply:
Now both have these sections:

```
       ? Row Ordering For In-Memory Tables
       ? Result Ordering For Database Tables
```